File: BE/API/Models/Cobranza.cs
namespace API.Models;

public class Cobranza
{
    public Guid Id { get; set; }
    public DateTime FechaLiquidacion { get; set; }
    public double Total { get; set; }
    public Cliente Cliente {get; set;}
    // public ICollection<DetalleCobranza> DetalleCobranza { get; set; }
    // public Cobranza()
    // {
    //     DetalleCobranza = new List<DetalleCobranza>();
    // }
}

File: FE/src/interfaces/cobranza.ts
import { Cliente } from "./cliente";
import { DetalleCobranza } from "./detalleCobranza";

export interface Cobranza {
    id: string,
    fechaLiquidacion: Date,
    total: number,
    cliente: Cliente,
    detalleCobranza: DetalleCobranza[]
}
